nvflare/private/fed/server/cmd_utils.py

```diff
@@ -134,11 +140,9 @@ def validate_command_targets(self, conn: Connection, args: List[str]) -> str:
         return ""
 
     def must_be_project_admin(self, conn: Connection, args: List[str]):
-        if args[0] == AdminCommandNames.ADMIN_CHECK_STATUS and len(args) == 1:
-            args.extend(["server"])
         role = conn.get_prop(ConnProps.USER_ROLE, "")
-        if role != "project_admin":
-            conn.append_error("Not authorized as project_admin.", meta=make_meta(MetaStatusValue.NOT_AUTHORIZED))
+        if role not in ["project_admin", "super"]:
```

Comment:
why do we need to introduce another super user "super"?

Reply:
Yeah, this was from old days that the admin console was generic and used the role name "super" in non-secure mode. I guess we could change it to "project_admin" since the admin console now is FLARE specific.

Reply:
changed to use project_admin only.